<ui version="4.0" >
 <class>OptionsWidget</class>
 <widget class="QWidget" name="OptionsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="toolTip" >
      <string>This list shows the active draw modes.
You can select multiple modes at the same time.</string>
     </property>
     <property name="statusTip" >
      <string/>
     </property>
     <property name="currentIndex" >
      <number>4</number>
     </property>
     <property name="usesScrollButtons" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="general" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>462</width>
        <height>685</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QCheckBox" name="fullscreen" >
         <property name="toolTip" >
          <string>Display OpenFlipper in fullscreen mode</string>
         </property>
         <property name="statusTip" >
          <string>Display OpenFlipper in fullscreen mode</string>
         </property>
         <property name="text" >
          <string>Show Fullscreen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="splash" >
         <property name="toolTip" >
          <string>Show splash screen on startup</string>
         </property>
         <property name="statusTip" >
          <string>Show splash screen on startup</string>
         </property>
         <property name="text" >
          <string>Show Splash-Screen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="toolbox" >
         <property name="toolTip" >
          <string>Show the toolbox for plugin widgets</string>
         </property>
         <property name="statusTip" >
          <string>Show the toolbox for plugin widgets</string>
         </property>
         <property name="text" >
          <string>Show Toolbox (CTRL+T)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="logger" >
         <property name="toolTip" >
          <string>Show the Logger window at the bottom</string>
         </property>
         <property name="statusTip" >
          <string>Show the Logger window at the bottom</string>
         </property>
         <property name="text" >
          <string>Show Logger (CTRL+L)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enableLogFile" >
         <property name="toolTip" >
          <string>Enable the logfile</string>
         </property>
         <property name="statusTip" >
          <string>Enable the logfile</string>
         </property>
         <property name="text" >
          <string>Logfile enabled</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="paths" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>462</width>
        <height>685</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Paths</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Logfile:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="logFile" >
         <property name="toolTip" >
          <string>filename (including path) of the logfile</string>
         </property>
         <property name="statusTip" >
          <string>filename (including path) of the logfile</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="viewer" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>462</width>
        <height>677</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Viewer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string> Mouse Wheel </string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout" >
          <property name="fieldGrowthPolicy" >
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>MouseWheel Zoomfactor:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="wZoomFactor" >
            <property name="maximumSize" >
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Set the factor to control the zooming with the mouse-wheel</string>
            </property>
            <property name="statusTip" >
             <string>Set the factor to control the zooming with the mouse-wheel</string>
            </property>
            <property name="inputMask" >
             <string>9.99; </string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>MouseWheel Zoomfactor (shift):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="wZoomFactorShift" >
            <property name="maximumSize" >
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Set the factor to control the zooming with the mouse-wheel while the shift key is pressed</string>
            </property>
            <property name="statusTip" >
             <string>Set the factor to control the zooming with the mouse-wheel while the shift key is pressed</string>
            </property>
            <property name="inputMask" >
             <string>9.99; </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string> DrawMode </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7" >
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8" >
            <item>
             <widget class="QLabel" name="label_7" >
              <property name="toolTip" >
               <string>This list shows the draw modes OpenFlipper uses at startup.
You can select multiple modes at the same time.</string>
              </property>
              <property name="text" >
               <string>Startup DrawModes:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="availDrawModes" >
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>300</height>
               </size>
              </property>
              <property name="toolTip" >
               <string>This list shows the draw modes OpenFlipper uses at startup.
You can select multiple modes at the same time.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string> Other </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QCheckBox" name="restrictFPS" >
              <property name="toolTip" >
               <string>restrict maximum framerate</string>
              </property>
              <property name="statusTip" >
               <string>restrict maximum framerate</string>
              </property>
              <property name="text" >
               <string>Restrict Framerate</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="FPS" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="toolTip" >
               <string>maximum framerate</string>
              </property>
              <property name="statusTip" >
               <string>maximum framerate</string>
              </property>
              <property name="minimum" >
               <number>1</number>
              </property>
              <property name="maximum" >
               <number>200</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8" >
              <property name="text" >
               <string>FPS</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="backfaceCulling" >
            <property name="toolTip" >
             <string>Enable backface culling</string>
            </property>
            <property name="statusTip" >
             <string>Enable backface culling</string>
            </property>
            <property name="text" >
             <string>Backface culling</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" >
            <item>
             <widget class="QPushButton" name="backgroundButton" >
              <property name="text" >
               <string>Default background color</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="key" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>462</width>
        <height>677</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Key Bindings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6" >
       <item>
        <widget class="QTreeWidget" name="keyTree" >
         <column>
          <property name="text" >
           <string>1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Updates" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>462</width>
        <height>677</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Updates</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3" >
           <item>
            <widget class="QLabel" name="label_4" >
             <property name="text" >
              <string>URL for updates</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6" >
             <property name="text" >
              <string>Password</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4" >
           <item>
            <widget class="QLineEdit" name="updateURL" />
           </item>
           <item>
            <widget class="QLineEdit" name="updateUser" />
           </item>
           <item>
            <widget class="QLineEdit" name="updatePass" />
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" >
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" >
           <item>
            <widget class="QPushButton" name="checkUpdateButton" >
             <property name="text" >
              <string>Check</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="updateButton" >
             <property name="text" >
              <string>Update</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>198</width>
             <height>27</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" >
         <item>
          <widget class="QLabel" name="statusLabel" >
           <property name="text" >
            <string>Check for Updates?</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>13</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="updateList" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Debug" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>462</width>
        <height>685</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Debug</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" >
         <item>
          <widget class="QCheckBox" name="slotDebugging" >
           <property name="toolTip" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Arial'; font-size:11pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Print information about plugins calling specific slots&lt;br />Currently supported slots:&lt;br />- updateView()&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">- updated_objects( id )&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="statusTip" >
            <string>Print information about plugins calling specific slots</string>
           </property>
           <property name="text" >
            <string>Enable slot debugging</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>340</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
